GitTfsTest/BaseTest.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Sep.Git.Tfs.Test
+{
+    public class BaseTest
+    {
+        public BaseTest()
+        {
+            if (!System.Diagnostics.Debugger.IsAttached)
+            {
+                System.Diagnostics.Trace.Listeners.Clear();
+            }
```

Comment:
Is there a way to do this that doesn't require every test to subclass `BaseTest`, or do anything at all? It'd be nice to be able to add new test classes without worrying about including this bit of setup.

Looking around a bit, it looks like the answer to my question is [no](http://stackoverflow.com/questions/12379949/how-to-run-setup-code-only-once-in-an-xunit-net-test), or not really. A static initializer might work, but a static initializer might not run often enough(?) and might run at the wrong time, too. It looks like there's an `IUseFixture` thing that would make it so that the test classes derive from an interface instead of a class, which is marginally better, but also harder to look at and understand.

So consider this comment a rant that you don't need to do anything about. Unless you have a good idea of how to make it better.